.storybook/typographyRenderers.tsx

```diff
@@ -4,24 +4,26 @@ import {
 	headlineObjectStyles,
 	textSansObjectStyles,
 } from '@guardian/source-foundations/src';
-import { fontWeightMapping, lineHeightMapping } from '@guardian/source-foundations/src/typography/data';
+import { fontWeightMapping } from '@guardian/source-foundations/src/typography/data';
 import type {
 	Category,
 	FontScaleFunction,
 	FontWeight,
-	LineHeight,
 } from '@guardian/source-foundations/src/typography/types';
 
 type FontFunctions = {
 	[key in Category]: FontScaleFunction;
 };
 
-interface Props {
+interface FontStylesRendererProps {
 	fontName: string;
 	fontStyles: FontFunctions;
 }
 
-export const FontStylesRenderer = ({ fontName, fontStyles }: Props) => {
+export const FontStylesRenderer: React.FC<FontStylesRendererProps> = ({
```

Comment:
In Source / DCR we prefer not to use `React.FC`, details on why are in here if you're interested: https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript

Reply:
👍 makes sense - will change back